dandi/support/bids/validator.py

```diff
@@ -453,7 +467,7 @@ def validate_all(
 
 def write_report(
     validation_result,
-    report_path="{logdir}/bids-validator-report_{datetime}-{pid}.log",
+    report_path="/var/tmp/bids-validator/report_{datetime}-{pid}.log",
```

Comment:
I suggest still using a variable for the log base directory, but set it to `$TMPDIR` (falling back to `/tmp`) by default.

Reply:
Again, this is for upstream usage, we wrap this command and pass the DANDI log directory via appdirs. `/tmp/` is tricky as it's ephemeral on some systems, and users might not know how their system handles it. `/var/tmp` is the safer bet.

Compare:
https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s15.html
to
https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch03s18.html
Particularly:
> Programs must not assume that any files or directories in /tmp are preserved between invocations of the program.

is pretty much a non-starter for logging.